jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -0,0 +1,255 @@
+
+The following settings are found by clicking the **Project Settings** button within your project. At this time, both configurations and triggers are limited to GitLab and GitHub App integrations.
+
+[#people]
+=== People
+
+Project roles give control over which users have access to which projects within an organization. This enables teams to have limited access to only their projects, while managers and others can have broader organizational access. The access options are:
+
+* Admin: Read and write access to the project and all settings and ability to manage other users' access.
+* Contributor: Read and write access to the project and some settings.
+* Viewer: Read only access to the project and some settings.
+
+For a complete list of permissions, see the xref:roles-and-permissions-overview#[Roles and permissions overview] page.
+
+[#configuration]
+=== Configuration
+
+You can add or delete a configuration source for your project. If you followed the steps above to connect GitHub, a GitHub configuration source has been automatically added for you.
+
+Once you define a configuration source, you can set up a trigger that points to that configuration.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-configuration.png[Configuration setup page]
+
+[#triggers]
+=== Triggers
+
+**The scheduled pipelines feature is not currently available for use with GitHub App integrations.** Triggers are described below, including how to use filters to trigger pipelines based on certain conditions.
+
+Add a trigger that specifies which configuration source starts a pipeline. If you followed the steps above to connect GitHub, a trigger set with GitHub as the configuration source has been automatically added for you.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-triggers.png[Trigger setup page]
+
+Triggers and trigger rules determine how CircleCI handles events from the source of change, in this case, GitHub.
+
+When a trigger is created, CircleCI registers a webhook with GitLab. Push events from GitLab are sent to CircleCI. CircleCI then uses the event data to determine _if_ a pipeline should run, and if so, _which_ pipeline should be run.
+
+In addition to a configuration source, each trigger includes the webhook URL, and in this scenario, a CircleCI-created GitLab token. The webhook URL and GitLab token are used to securely register the webhook within GitLab in order to receive push events from your GitLab repo.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-edit-trigger.png[Trigger details]
+
+**Trigger filters** allow you to determine when a trigger should initiate a build based on the parameters provided by Gitlab’s webhook. CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
```

Comment:
No custom filters at this time. I'm not sure about the only build merge requests option 🤔 I'm not sure where that is applied.